src/main/java/Duke.java

```diff
@@ -1,10 +1,46 @@
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        System.out.println("-------------------------------");
+        System.out.println("Hello! I'm Duke\n" +
+                "What can I do for you?");
+        System.out.println("-------------------------------");
+
+        String echo = " ";
+        ArrayList<String> addedList = new ArrayList<>();
+
+        Scanner input = new Scanner(System.in);
+        while (!echo.equals("bye")) {
+            echo = input.nextLine();
+            if(!echo.equals("bye") && !echo.equals("list")){
```

Comment:
Perhaps consider scenarios where input is similar (e.g. BYE / LIST), should this be interpreted as an "action" for duke?

Reply:
does it mean need to add Bye, List, Done as command class as an "action"